packages/react-vis/package.json

```diff
@@ -79,7 +74,6 @@
     "eslint-plugin-no-only-tests": "^2.0.1",
     "eslint-plugin-prettier": "^2.6.2",
     "eslint-plugin-react": "^7.11.1",
-    "husky": "^1.1.2",
```

Comment:
and we still need `huskey` listed as a dependency somewhere

Reply:
yes I moved it to the root package.json file